ietf/rfc/rfc748/index.md

```diff
@@ -35,29 +35,49 @@ NIC 44125                                                   1 April 1978
 
                       TELNET RANDOMLY-LOSE Option
 
-1.  Command name and code.
+1.  Nome e código do comando.
 
    RANDOMLY-LOSE        256
 
-2.  Command meanings.
+2.  Significado do comando (O que faz).
 
    IAC WILL RANDOMLY-LOSE
 
       The sender  of this command  REQUESTS  permission  to, or confirms
       that it will, randomly lose.
```

Comment:
```The sender  of this command  REQUESTS  permission  to, or confirms
        that it will, randomly lose.```

O original em inglês pode ser removido na minha opinião